src/Core__Array.resi

```diff
@@ -101,11 +141,34 @@ external setUnsafe: (array<'a>, int, 'a) => unit = "%array_unsafe_set"
 let findIndexOpt: (array<'a>, 'a => bool) => option<int>
 let reverse: array<'a> => array<'a>
 let filterMap: (array<'a>, 'a => option<'b>) => array<'b>
+
+/**
+  `fromInitializer(~length, f)`
+
+  Creates an array of length `length` initialized with the value returned from `f ` for each index.
+
+  ```res example
+  Array.make(~length=3, i => i + 3) == [3, 4, 5]
+  ```
+*/
+let keepSome: array<option<'a>> => array<'a>
```

Comment:
This doesn't seem to be the right docstring?

Reply:
I actually just forgot to save the file after the last edit :facepalm: Fixed in https://github.com/rescript-association/rescript-core/pull/49/commits/b80a80f10b4c925d6f91fe06c7493a876efc64a8